src/main/java/com/example/coursework2/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.example.coursework2;
+
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+@Service
+public class ExaminerServiceImpl implements ExaminerService {
+
+
+    public final JavaQuestionService javaQuestionService;
+
+    public ExaminerServiceImpl(JavaQuestionService javaQuestionService) {
```

Comment:
```suggestion
    public ExaminerServiceImpl(QuestionService javaQuestionService) {
```
инъекцию надо делать по интерфейсу